### ISSUE TITLE
Style resources on Host your project page

### ISSUE THREAD

**Issue by thetwopct** [lfph/lfph.io#45]
/join/host-your-project/

![20200719181152-Host your project - Linux Foundation Public Health](https://user-images.githubusercontent.com/10615884/87880663-5ba10c00-c9eb-11ea-977a-e73e1cf667ec.jpg)

**Comment by thetwopct** [comment]
I've updated the styling on this page, and embedded the images in to WP instead of pulling from Google Drive.

https://dev-lfph.pantheonsite.io/join/host-your-project/

I had to crop all the images manually as they weren't centered in their frames.

I've noticed some of the images have faint diagonal lines in them - so lets be sure we have usage rights. We may want to get the design team to make some custom icons to inject some colour to the page (perhaps the ones on [CNCF](https://dev-cncfci.pantheonsite.io/about/join/#marketing) could be re-adapted with blues instead of pinks?)
